lib/services/role_test.go

```diff
@@ -4234,6 +4234,73 @@ func TestHostUsers_getGroups(t *testing.T) {
 	}
 }
 
+func TestHostUsers_HostSudoers(t *testing.T) {
```

Comment:
I'd like a test with trait substitution in here, to make sure it's doing what we want.

Reply:
if we are going to include `deny` logic, should test that as well